packages/app/src/components/VisualizationOptions/Options/Outliers.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import i18n from '@dhis2/d2-i18n'
+import { Checkbox, FieldSet, Help, Legend } from '@dhis2/ui'
+
+import { sGetUiOptions } from '../../../reducers/ui'
+import { acSetUiOptions } from '../../../actions/ui'
+import {
+    tabSectionOptionToggleable,
+    tabSectionOption,
+    tabSectionTitle,
+} from '../styles/VisualizationOptions.style.js'
+import OutlierDetectionMethod from './OutlierDetectionMethod'
+import ExtremeLines from './ExtremeLines'
+import styles from '../styles/Outliers.module.css'
+
+const ENABLED_PROP = 'enabled'
+const METHOD_PROP = 'method'
```

Comment:
Looking at [the ticket for adding backend support for the outliers AO option](https://jira.dhis2.org/browse/DHIS2-10079) this should be changed to:

```suggestion
const METHOD_PROP = 'outlierMethod'
```

Reply:
Good find!

Reply:
BTW I see Jan's PR has the same constants, maybe import them from analytics?
See https://github.com/dhis2/analytics/pull/831/files/b889bd63c96f25cae7635c6e4f2dc3ea0c15190b..7209db8591c105f53183144dc012615adddb85a0#diff-70166e43854b44e7e94c473a865ff33fe427f9b2d0184a10b62253e36a0144e5